components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Settings.php

```diff
@@ -95,111 +101,150 @@ public function tab( $tabs, $pod, $addtl_args ) {
+		);
+
+		$type = $pod['type'];
+
+		switch ( $type ) {
+			case 'post_type':
+				$options['pods-pfat'] = array(
+					'pfat_enable'           => array(
+						'label'             => __( 'Enable Automatic Pods Templates for this Pod?', 'pods' ),
+						'help'              => __( 'When enabled you can specify the names of Pods Templates to be used to display items in this Pod in the front-end.', 'pods' ),
+						'type'              => 'boolean',
+						'default'           => false,
+						'dependency'        => true,
+						'boolean_yes_label' => '',
+					),
+					'pfat_run_outside_loop' => array(
+						'label'             => __( 'Execute Auto Template outside of the WordPress loop? (advanced)', 'pods' ),
+						'help'              => __( 'When enabled, the template will be executed whenever the specified filter is called.', 'pods' ),
+						'type'              => 'boolean',
+						'default'           => false,
+						'depends-on'        => array( 'pfat_enable' => true ),
+						'boolean_yes_label' => '',
+					),
+					'pfat_single'           => array(
+						'label'      => __( 'Single view template', 'pods' ),
+						'help'       => __( 'Name of Pods template to use for single item view.', 'pods' ),
+						'type'       => 'text',
+						'default'    => false,
+						'depends-on' => array( 'pfat_enable' => true ),
+					),
+					'pfat_append_single'    => array(
+						'label'      => __( 'Single Template Location', 'pods' ),
+						'help'       => __( 'Whether the template will go before, after or in place of the post content.', 'pods' ),
+						'depends-on' => array( 'pfat_enable' => true ),
+					),
+					'pfat_filter_single'    => array(
+						'label'      => __( 'Single Template Filter', 'pods' ),
+						'help'       => __( 'Which filter to use for single views.', 'pods' ),
+						'default'    => 'the_content',
+						'type'       => 'text',
+						'depends-on' => array( 'pfat_enable' => true ),
+					),
+					'pfat_archive'          => array(
+						'label'      => __( 'Archive view template', 'pods' ),
+						'help'       => __( 'Name of Pods template to use for use in this Pods archive pages.', 'pods' ),
+						'type'       => 'text',
+						'default'    => false,
+						'depends-on' => array( 'pfat_enable' => true ),
+					),
+					'pfat_append_archive'   => array(
+						'label'      => __( 'Archive Template Location', 'pods' ),
+						'help'       => __( 'Whether the template will go before, after or in place of the post content.', 'pods' ),
+						'depends-on' => array( 'pfat_enable' => true ),
+					),
+					'pfat_filter_archive'   => array(
+						'label'      => __( 'Archive Template Filter', 'pods' ),
+						'help'       => __( 'Which filter to use for archives.', 'pods' ),
+						'default'    => 'the_excerpt',
+						'type'       => 'text',
+						'depends-on' => array( 'pfat_enable' => true ),
+					),
+				);
+				break;
+
+			case 'taxonomy':
+			case 'user':
+			case 'comment':
+				$options['pods-pfat'] = array(
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: f98b46ff9bdb53c153f577b5bb5322aa</sub>